core/designsystems/src/main/java/at/marki/moviedb/core/designsystems/components/MovieListElement.kt

```diff
@@ -36,6 +36,7 @@ private const val POSTER_WIDTH = 60
 @Composable
 fun MovieListElement(
     isFavorite: Boolean,
+    //REVIEW: would have expected some reduced ui model here
```

Comment:
We only ever do this when we heavily modify the Data (e.g. for performance gains). Will think about this approach though.